src/components/common/Error.tsx

```
@@ -0,0 +1,22 @@
+import Image from 'next/image';
+
+interface ErrorProps {
+  message?: string;
+}
+
+const Error = ({ message = '앗! 에러가 발생했어요' }: ErrorProps) => {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)